package.json

```diff
@@ -8,6 +8,17 @@
     "url": "https://github.com/arc-repos/arc-example-login-flow"
   },
   "dependencies": {
-    "@architect/architect": "^4.1.0"
+    "@architect/architect": "^4.4.9",
+    "@architect/functions": "git+https://github.com/arc-repos/arc-functions.git"
```

Comment:
Hrm yes, we need `@architect/functions`, so we need to keep that line. Since the code has been merged though we can update the dependency to the stable version, rather than having to use github.